stack-cmbm/templates/default/puma.conf.erb

```diff
@@ -0,0 +1,3 @@
+<% @apps.each do |app| -%>
```

Comment:
@baccenfutter I guess this template can go? Nowhere used, and puma has been removed.